jenkins/Pipeline.gy

```diff
@@ -64,11 +64,9 @@ pipeline {
     }
     post {
         always {
-            // Remove the pod which we just created and ran; remove any
-            // dangling containers; and then remove any dangling images.
-            sh 'podman pod rm -f ${PB_POD_NAME}'
-            sh 'podman container prune -f'
-            sh 'podman image prune -f'
+            // Remove the pod which we just created and ran, and perform a
+            // general clean up of podman related resources.
+            sh 'jenkins/podman-cleanup ${PB_POD_NAME}'
         }
```

Comment:
I'm not a fan of having lots of little scriptlets floating around, particularly ones which have specialized uses.

If we think that `podman-cleanup` is of general use, then perhaps we should put it in `.../utils`.  Otherwise, perhaps something like this would be better:
```
        always {
            // Remove the pod which we just created and ran; remove any
            // dangling containers; and then remove any dangling images.
            sh 'podman pod rm -f ${PB_POD_NAME} || true'
            sh 'podman container prune -f || true'
            sh 'podman image prune -f || true'
        }
```